packages/models/src/lib/implementation/schemas.ts

```diff
@@ -7,8 +7,10 @@ import {
 } from './limits';
 import { filenameRegex, slugRegex } from './utils';
 
-export const primitiveValueSchema = z.union([z.string(), z.number()]);
-export type PrimitiveValue = z.infer<typeof primitiveValueSchema>;
+export const tableCellValueSchema = z
+  .union([z.string(), z.number(), z.boolean(), z.null()])
+  .default(null);
```

Comment:
I added a default to `null` because I ran into en issue where there where more columns described then present in a row. This avoided bigger refactoring and should still be open for formatting logic.

## Error


### LH Audit Data:

```json
"csp-xss": {
  "id": "csp-xss",
  "title": "Ensure CSP is effective against XSS attacks",
  "description": "A strong Content Security Policy (CSP) significantly reduces the risk of cross-site scripting (XSS) attacks. [Learn how to use a CSP to prevent XSS](https://developer.chrome.com/docs/lighthouse/best-practices/csp-xss/)",
  "score": 1,
  "scoreDisplayMode": "informative",
  "details": {
    "type": "table",
    "headings": [
      {
        "key": "description",
        "valueType": "text",
        "subItemsHeading": {
          "key": "description"
        },
        "label": "Description"
      },
      {
        "key": "directive",
        "valueType": "code",
        "subItemsHeading": {
          "key": "directive"
        },
        "label": "Directive"
      },
      {
        "key": "severity",
        "valueType": "text",
        "subItemsHeading": {
          "key": "severity"
        },
        "label": "Severity"
      }
    ],
    "items": [
      {
        "severity": "High",
        "description": "No CSP found in enforcement mode"
      }
    ]
  }
},
```

### Error Message

```bash
Error: - Plugin Lighthouse (lighthouse) produced the following error:
  - 
Audit csp-xss failed parsing details: 
Parsing details table failed: 
Raw data:
 {"rows":[{"severity":"High","description":"No CSP found in enforcement mode"}]}
[
  {
    "code": "invalid_union",
    "unionErrors": [
      {
        "issues": [
          {
            "code": "invalid_type",
            "expected": "array",
            "received": "object",
            "path": [
              "rows",
              0
            ],
            "message": "Expected array, received object"
          }
        ],
        "name": "ZodError"
      },
      {
        "issues": [
          {
            "code": "invalid_union",
            "unionErrors": [
              {
                "issues": [
                  {
                    "code": "invalid_type",
                    "expected": "string",
                    "received": "undefined",
                    "path": [
                      "rows",
                      0,
                      "directive"
                    ],
                    "message": "Required"
                  }
                ],
                "name": "ZodError"
              },
              {
                "issues": [
                  {
                    "code": "invalid_type",
                    "expected": "number",
                    "received": "undefined",
                    "path": [
                      "rows",
                      0,
                      "directive"
                    ],
                    "message": "Required"
                  }
                ],
                "name": "ZodError"
              },
              {
                "issues": [
                  {
                    "code": "invalid_type",
                    "expected": "boolean",
                    "received": "undefined",
                    "path": [
                      "rows",
                      0,
                      "directive"
                    ],
                    "message": "Required"
                  }
                ],
                "name": "ZodError"
              },
              {
                "issues": [
                  {
                    "code": "invalid_type",
                    "expected": "null",
                    "received": "undefined",
                    "path": [
                      "rows",
                      0,
                      "directive"
                    ],
                    "message": "Required"
                  }
                ],
                "name": "ZodError"
              }
            ],
            "path": [
              "rows",
              0,
              "directive"
            ],
            "message": "Invalid input"
          }
        ],
        "name": "ZodError"
      }
    ],
    "path": [],
    "message": "Invalid input"
  }
]
```

Reply:
Sure, makes sense to me, thanks for the explanation :slightly_smiling_face: